src/index.js

```diff
@@ -85,6 +85,7 @@ class Migrate extends Command {
       await this.confirmTransforms(sdk);
       this.log.info('Migration complete!');
     } else {
+      if (sdk) this.log.warn('Make sure your SDK is upgraded to the latest version!');
```

Comment:
Is it worth interpolating the version? From the migration class?

Reply:
We could, but as of now I made those versions be the next major versions which do not exist for Java or Ruby yet

Reply:
Yeah I saw, I think that is perfect. I liked that because it's clear which version _won't_ trigger the deprecation message